src/components/Books.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Books = () => (
+  <>
+    <div className="books-container" />
+    <h2>ADD NEW BOOK</h2>
+    <form>
+      <input type="text" placeholder="Book title" />
+      <select disabled>
+        <option value="category">Category</option>
+      </select>
+      <button type="button">ADD BOOK</button>
+    </form>
+  </>
+);
+
+export default Books;
```

Comment:
- [ ] You should have a component that displays a list of books, and a component that displays a single book, for example `BookList` and `Book`, kindly implement this. 😄

Reply:
- [ ] Your component that lists the books, should pass the data to each book as a prop, for example:
    - `<Book title={book[0]} />`, I would recommend you to create a list of default books on your `Books` state 🚀